doc/index.rst

```diff
@@ -17,8 +17,9 @@ state-of-the-art spike sorters, post-process and curate the output, compute qual
 
     **New features under construction!** 🚧🚧🚧: after the 0.100.0 release (and related bug fixes), the next release will contain
     a major API improvement: the :code:`SortingAnalyzer`. To read more about this, checkout the
-    `enhancement proposal <https://github.com/SpikeInterface/spikeinterface/issues/2282>`_.
-    Please refer to the stable documentation `here <https://spikeinterface.readthedocs.io/en/0.100.1>`_
+    `enhancement proposal <https://github.com/SpikeInterface/spikeinterface/issues/2282>`_. Learn how to update your code
+    `here <how_to/waveform_extractor_to_sorting_analyzer.html>`_ and read more about the
+    :code:`SortingAnalyzer` `here <modules/postprocessing.html>`_.
```

Comment:
The SortingAnalyzer itself lives in core. So not sure about this line. I think linking the `how_to` is great, though.